lib/github-auth.js

```diff
@@ -120,7 +120,7 @@ function sendTokenToAllServers(token) {
         strictSSL: false,
       };
       request(options, function(err, res, body) {
-        if (err != null) { return reject('Posting the GitHub user token failed: ' + err.stack); }
+        if (err != null) { return reject(err); }
```

Comment:
Could do `return reject(new Error('Posting the GitHub user token failed: ' + err.message))` or something similar. Up to you.

Reply:
Yea… I thought about that. This is called in a loop, and the calling code already has something really similar to that. It feels redundant to prefix it twice.